docs/sidebar.yml

```diff
@@ -3,6 +3,10 @@ sidebar:
       url: blog/index.html
     - title: Docs
       url: docs/index.html
+    - title: Reference
```

Comment:
Should we call this "Language Reference"? I'm guessing you're following suit on what @odersky is doing with his docs. Did we decide to change this yesterday?